### ISSUE TITLE
App Certification failed on one of our apps

### ISSUE THREAD

**Issue by libreliodev** [libreliodev/windows8#83]
There is a specific reason that we will easily fix: "Error Found: The app manifest test detected the following errors: ◦The contrast ratio of the background color and foreground text color is 1,03. This ratio should be greater than or equal to 1,5."

But other issues are also listed, which should exist in other apps, and have not prevented the other apps from being certified.

@dd1804 : Any idea why?

Windows App Certification Kit - Test Results

App name: NivalesMdias.WiderMagazine
App version: 1.0.0.23
App publisher: Nivéales Médias
OS Version: Microsoft Windows 8 (6.2.9200.0)
Kit Version: 2.2
Report time: 14/06/2013 12:25:11

---

Overall result: FAILED 

---

Crashes and hangs test 

PASSED 
App launch tests

PASSED 
Crashes and hangs

App manifest compliance test 

FAILED 
App manifest
•
Error Found: The app manifest test detected the following errors: ◦The contrast ratio of the background color and foreground text color is 1,03. This ratio should be greater than or equal to 1,5.

•
Impact if not fixed: Apps that violate the Windows Store certification requirements can’t be submitted to the Windows Store. 

•
How to fix: App manifest must include valid entries for all required fields, please modify the manifest entries listed above. See link below for more information:
Certification requirements  

Windows security features test 

PASSED 
Binary analyzer

PASSED 
Private Code Signing

Supported API test 

FAILED 
Supported APIs
•
Error Found: The supported APIs test detected the following errors: ◦API GetModuleFileNameW in kernel32.dll is not supported for this application type. MuPDFWinRT.dll calls this API.
◦API GetModuleHandleW in kernel32.dll is not supported for this application type. MuPDFWinRT.dll calls this API.
◦API LoadLibraryExW in kernel32.dll is not supported for this application type. MuPDFWinRT.dll calls this API.
◦API LoadLibraryW in kernel32.dll is not supported for this application type. MuPDFWinRT.dll calls this API.

•
Impact if not fixed:  Using an API that is not part of the Windows SDK for Windows Store apps violates the Windows Store certification requirements.  

•
How to fix:  Review the error messages to identify the API that is not part of the Windows SDK for Windows Store apps. Please note, C++ apps that are built in a debug configuration will fail this test even if it only uses APIs from the Windows SDK for Windows Store apps. See the link below for more information: 
 Alternatives to Windows APIs in Windows Store apps.   

Performance test 

PASSED 
Bytecode generation

PASSED 
Optimized Binding References

PASSED 
Performance launch

PASSED 
Performance suspend

App manifest resources test 

PASSED 
App resources validation

PASSED 
Branding validation

Debug configuration test 

FAILED 
Debug configuration
•
Error Found: The debug configuration test detected the following errors: ◦The binary MuPDFWinRT.dll is built in debug mode.
◦The binary LibrelioMagazine.exe is built in debug mode.

•
Impact if not fixed:  Windows Store doesn’t allow a debug version of an app.  

•
How to fix:  Please make sure the app isn’t linking to any debug versions of a framework, and it is built with release configuration. If this is a managed app please make sure you have installed the correct version of .NET framework.  

File encoding 

PASSED 
UTF-8 file encoding

Direct3D feature level support 

PASSED 
Direct3D feature level support

App Capabilities test 

PASSED 
Special Use Capabilities

Windows Runtime metadata validation 

PASSED 
ExclusiveTo attribute test

PASSED 
Type location test

PASSED 
Type name case-sensitivity test

PASSED 
Type name correctness test

PASSED 
General metadata correctness test

PASSED 
Properties test
